### ISSUE TITLE
Press more buttons at one time doesn't work?

### ISSUE THREAD

**Issue by Klaus1189** [libretro/RetroArch#10483]
I play Cruis'n USA for N64 and want to drive the police car, but in order to do this I need to press C up, C down and C left at the same time then the police car appears at the second car from the right.

On my original N64 it works, but using the Retro-Bit Tribute 64 USB controller on my PC using RetroArch doesn't work. I think it doesn't recognize the three button presses somehow.
If I need to configure something, please let me know.
Playing other games is fun ;-) So playing Super Mario 64 works fine and all buttons are working.

If you need more info, please let me know.

**Comment by i30817** [comment]
This kind of thing is usually a device or bios/driver limitation.

**Comment by barbudreadmon** [comment]
Yes, probably some kind of ghosting issue, what's surprising is that i thought it only affected keyboards. Is your gamepad somehow recognized as a keyboard on your computer ?

**Comment by sonninnos** [comment]
Could it just be the default setting of "Independent C-button Controls" being false in Mupen64, which absolutely makes it impossible to press up and down at the same time due to them being at the same analog axis, and not actual buttons.

**Comment by Klaus1189** [comment]
Sorry for late reply, it took some more testing time.
@sonninnos: Thank you, that is exactly the setting I needed. :)
Another USB Controller does work and my keyboard does work now as well.

But nevertheless these N64 controllers still don't work, is there anything I can do? I have problems with mapping too on this game too, car brakes when pressing B, which is supposed to change the song. I think I check the mapping options.
